gno.land/pkg/sdk/vm/builtins.go

```diff
@@ -91,4 +91,10 @@ func (bnk *SDKBanker) RemoveCoin(b32addr crypto.Bech32Address, denom string, amo
 	if err != nil {
 		panic(err)
 	}
+	// TODO: once TotalCoin is implemented, we need to deduct the TotalCoins beside send coin to ZeroAddress (?)
+	zeroAddr := std.ZeroAddress()
+	_, err = bnk.vmk.bank.AddCoins(bnk.ctx, zeroAddr, std.Coins{std.Coin{denom, amount}})
```

Comment:
why adding the removed coins to the zero address?

Reply:
@moul 
This is because of the burn process definition: Should `send` coins from burner to zero address, beside removing that amount of coins from Total Supply.
Adding to ZeroAddress making everyone can see how many coins is burned, and how many coins zero address currently holds. Deducting coins from total supply is the target of burning theory.